sshd-core/src/main/java/org/apache/sshd/server/forward/TcpipServerChannel.java

```diff
@@ -260,6 +260,20 @@ protected void handleChannelOpenFailure(OpenFuture f, Throwable problem) {
 
     @Override
     public CloseFuture close(boolean immediately) {
+
+        if (!immediately && (out != null)) {
+            try {
+                if (log.isDebugEnabled()) {
+                    log.debug("Closing ChannelOutputStream: {}", TcpipServerChannel.this);
+                }
+                out.close();
+            } catch (IOException ignored) {
+                if (log.isErrorEnabled()) {
+                    log.error("Error closing ChannelOutputStream: {}", TcpipServerChannel.this, ignored);
+                }
+            }
+        }
```

Comment:
1. We never ask if ERROR enabled - only DEBUG and TRACE
2. If this is an ignored exception why log it as ERROR ? DEBUG or TRACE would seem more appropriate